ComponentFramework3.0/ComponentFramework/JSONFile.cpp

```diff
@@ -0,0 +1,86 @@
+#include "FileManager.h"
+#include <vector>
+#include "json.hpp"
+#include <fstream>
+#include <iostream>
+#include <string>
+#include "Scene0.h"
+#include "GameObject.h"
+#include "JSONFile.h"
+
+
+using json = nlohmann::json;
+
+using namespace GAME;
+using namespace MATH;
+
+
+
+GAME::JSONFile::JSONFile()
+{
+}
+
+GAME::JSONFile::~JSONFile()
+{
+}
+
+void GAME::JSONFile::OnWrite(std::vector<GameObject*> ScenceModelList)
+{
+	json j;
+
+	for (int i = 0; i < ScenceModelList.size(); i++)
+	{
+
+		j["object" + std::to_string(i)]["type"] = ScenceModelList[i]->GetMOdelName();
```

Comment:
use a constant instead of the raw string "object" as you could potentially miss spell it having to retype it so many times.